exercises/concept/welcome-to-tech-palace/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+There is an appliance store called "Tech Palace" nearby. The owner of the store recently installed a big display to use for marketing messages and to show a special greetings when customers scan their loyalty cards at the entrance. The display consists of lots of small LED lights and can show multiple lines of text.
```

Comment:
```suggestion
There is an appliance store called "Tech Palace" nearby.
The owner of the store recently installed a big display to use for marketing messages and to show a special greetings when customers scan their loyalty cards at the entrance.
The display consists of lots of small LED lights and can show multiple lines of text.
```

Mind breaking up the sentences one per line like this?

Reply:
👍